### ISSUE TITLE
Inconsistent information about hyperspace messaging

### ISSUE THREAD

**Issue by ThrawnCA** [endless-sky/endless-sky#7895]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the bug

After the hyperspace links to Rand are cut during the Free Worlds campaign, Alondo says that Net messages rely on those links, so all communications with the Syndicate are cut.

However, it is entirely possible for the player to receive mission callbacks in the form of Net messages, while in areas not connected to human space via hyperlinks. I was even able to receive the "FW Stack Core 1C" message from Freya while on the other side of the cut links (using a Jump Drive).

### Steps to Reproduce

1. Obtain at least one Jump Drive from Korath Palavrets.
2. Complete the Free Worlds campaign up to but not including "FW Syndicate Capture 1C".
3. Complete side mission "FW Stack Core 1B".
4. Immediately complete "FW Syndicate Capture 1C" and receive ""FW Pug 1".
5. Jump to Rand and wait for the "FW Stack Core 1B" message.

### Expected Behavior

If Net messages rely on hyperspace links, then "FW Stack Core 1C" should only be received once the player is back in a system reachable from human space.

### Operating System

Ubuntu Linux 22.04

### Game Source

GitHub Releases

### Game Version

0.9.16.1

**Comment by warp-core** [comment]
Sorry it's taken so long to get to this.
The steps to reproduce don't make sense to me.
"FW Stack Core 1B" is a mission offered from Rand, with the destination being Solace in the Pherkad system. There is an on offer conversation (which has no choices and simply accepts the mission), an on visit dialog, and an on complete dialog. The mission offers on landing. It is all but impossible to finish the first mission (either "FW Stack Core 1" or "FWC Stack Core 1") and not get this mission immediately.
Once you have completed this mission (as the reproduction steps direct) there are no more messages from this mission to receive. There is only "FW Stack Core 1C" which contains only a conversation that declines. And, once the Pug invasion starts and Rand is disconnected from human space, this mission cannot be received there.
Do you mean that the message in "FW Stack Core 1C" can be received in Syndicate space, despite the hyperspace relays in this region supposedly not being connected to the hyperspace relays in the Free Worlds and the rest of the Republic?

**Comment by ThrawnCA** [comment]
:D Sorry, it's been over two years since I opened this and I don't remember the exact mission codes. I'll take a look later and see if I made a typo.

Thanks for following up!

**Comment by ThrawnCA** [comment]
> Do you mean that the message in "FW Stack Core 1C" can be received in Syndicate space, despite the hyperspace relays in this region supposedly not being connected to the hyperspace relays in the Free Worlds and the rest of the Republic?

Yes, that's correct. I mistyped step 5 as 1B when it should have been 1C.

It's probably not the only possible example.

**Comment by Aradziem** [comment]
I think this is a similiar case (I'm in Coalition).

<img width="1366" height="768" alt="Image" src="https://github.com/user-attachments/assets/c560f6dd-1f72-4cd6-b017-99c3e6c486e9" />
